config.go

```diff
@@ -0,0 +1,5 @@
+package tigerblood
+
+type Config struct {
+	Credentials map[string]string
```

Comment:
This doesn't need to be a map, it could just be a slice of credentials.
```go
type Config struct {
    Credentials []credential
}
type credential {
    User string
    Token string
}
```
or something long those lines